webpack.config.node.js

```diff
@@ -105,6 +105,9 @@ var webpackConfig = {
 	plugins: [
 		// Require source-map-support at the top, so we get source maps for the bundle
 		new webpack.BannerPlugin( 'require( "source-map-support" ).install();', { raw: true, entryOnly: false } ),
+		new webpack.DefinePlugin( {
```

Comment:
@ehg I added `PROJECT_NAME` here, too. I didn't find a way how to use it in `sections.js` file. I know it doesn't work because it is rewritten with loader which doesn't care about the global Webpack config. I'm not sure how to fix it so I leave it for the future iteration.